arch/sim/src/sim/posix/sim_alsa.c

```diff
@@ -595,6 +595,11 @@ static int sim_audio_ioctl(struct audio_lowerhalf_s *dev, int cmd,
         }
         break;
 
+        case AUDIOIOC_SETPARAMTER:
+        {
+          audinfo("%s , arg: %s\n", __func__, (char *)arg);
+        } break;
```

Comment:
move `break;` to the new line

Reply:
yes, i got it. and fixed in https://github.com/apache/nuttx/pull/9693. 
please review, thanks